core/src/test/scala/unit/kafka/server/LogDirFailureTest.scala

```diff
@@ -168,6 +169,10 @@ class LogDirFailureTest extends IntegrationTestHarness {
   }
 
   def testProduceAfterLogDirFailureOnLeader(failureType: LogDirFailureType, quorum: String): Unit = {
+    if (isKRaftTest()) {
+      val value = configs.map(c => c.brokerId -> c.logDirs.contains(c.metadataLogDir))
+      logger.warn(s">>>>>> ${value.mkString(",")}")
+    }
```

Comment:
Did you mean to include this change? The logging format and lack of description seems odd.

Reply:
I thought I removed this one!

Reply:
will raise a minor pr to remove it. I removed it but didn't push the commit ops

Reply:
https://github.com/apache/kafka/pull/15371